packages/functions/src/sendEmail/https.ts

```diff
@@ -28,18 +34,43 @@ export const sendEmail = functions
       ) {
         throwUnauth();
       }
+      await getSMTPPreferences(organization);
 
       checkRequiredFields(email, ["to", "html", "subject"]);
 
+      // Get email preferences from organization info
+      const db = admin.firestore();
+      const orgSnap = await db
+        .doc(`${Collection.Organizations}/${organization}`)
+        .get();
+      const orgData = orgSnap.data() as OrganizationData;
+
       // add email to firestore, firing data trigger
-      await admin
+      const doc = admin
         .firestore()
         .collection(
           `${Collection.DeliveryQueues}/${organization}/${DeliveryQueue.EmailQueue}`
         )
-        .doc()
-        .set({ payload: email });
+        .doc();
+
+      checkRequiredFields(orgData, ["emailFrom"]);
```

Comment:
This should be handled by `smtpConfigured` flag

Reply:
Ok...I see...could we make it like this:
- check if `smtpConfigured`, if not: throw some `NoSMTPConfigured` error
- check if `emailFrom` exists, if not: throw some `NoEmailFrom` or something...

Reply:
No need for `checkRequired` fields since it's only one field: we can check `if (!orgData.emailFrom) throw Error(...)`